### DISCUSSION TITLE
I recently upgraded to the dev kit and I now cannot bulid my project and it gives me this error.

### DISCUSSION THREAD

**Discussion by SuperWasTaken** [CosmosOS/Cosmos#2795] [Q&A]
![image](https://github.com/CosmosOS/Cosmos/assets/145633746/731decf7-927e-4d4c-b6ae-c6112a2afefe)

**Comment by SuperWasTaken** [comment]
Yes I have the startup project configured it just refuses to bulid and gives me this error

**Reply by 9xbt** [reply]
Hmm, can you show me your csproj file?

**Reply by SuperWasTaken** [reply]
I was able to fix it by downgrading back to the user kit and transferring everything back
